hadoop-ozone/dist/src/shell/hdds/hadoop-functions.sh

```diff
@@ -1541,6 +1541,21 @@ function hadoop_translate_cygwin_path
   fi
 }
 
+## @description  Adds default GC parameters
+## @description  Only for server components and only if no other -XX parameters
+## @description  are set
+## @audience     private
+## @stability    evolving
+## @replaceable  yes
+function hadoop_add_default_gc_opts
+{
+  if [[ "${HADOOP_SUBCMD_SUPPORTDAEMONIZATION}" == true ]]; then
+    hadoop_debug "Appending default GC parameter to the HADOOP_OPTS"
```

Comment:
I think this should be logged only if `HADOOP_OPTS` is really being changed.

Reply:
good point. fixed.